pkgs/by-name/fi/firewalk/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchurl
+, libnet
+, libpcap
+, libdnet
+}:
+
+stdenv.mkDerivation rec {
+  pname = "firewalk";
+  version = "5.0";
+
+  src = fetchurl {
+    url = "http://web.archive.org/web/20060925223218/http://www.packetfactory.net/firewalk/dist/firewalk-5.0.tgz";
+    hash = "sha256-zVSNBm/SFVWqBHfeYxiHpH/jDnsbisXMrMg3m/Qw1ro=";
+  };
+
+  buildInputs = [ libnet libpcap libdnet ];
+
+  meta = with lib; {
+    description = "Gateway ACL scanner";
+    homepage = "http://packetfactory.openwall.net/projects/firewalk/";
+    platforms = platforms.linux;
+    license = licenses.bsd2;
+    maintainers = with maintainers; [  ];
```

Comment:
You need to add yourself as a maintainer

Reply:
Well, it's not strictly obligatory. It's written as an attribtute that [*should be set*](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes), but if you are very against it for some reason, it's okay.

Reply:
PRs definitely have been held up from maintainers not being set, so I'm just recommending from that perspective

Reply:
Sure, that's true. But I still think it's important to know.